acceptancetests/assess_destroy_model.py

```diff
@@ -68,7 +68,11 @@ def destroy_model(client, new_client):
             if not_found_error in e.stderr:
                 log.info("Model fully removed")
                 break
-            removed_error = b'model "{}" has been removed from the controller'.format(old_model)
+            # The message actually looks like `ERROR model "admin/test-tmp-env"
+            # has been removed from the controller...` - old_model only
+            # contains the model name, hence the weird partial quoting so that
+            # it matches.
+            removed_error = b'{}" has been removed from the controller'.format(old_model)
```

Comment:
Missing beginning double quote? Though I think the test will still work.
s/b'{}" has/b'"{}" has

Reply:
Perhaps I need an afternoon tea.  Makes more sense after re-reading the comment.

Reply:
After chatting I think it's too confusing, tweaked it to just include the username `admin` and the starting double quote instead.